### PR TITLE
dotCMS/core#16859 Add icons to edit mode selectors and match design

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: src/app/view/components/_common/dot-icon/dot-icon.component.ts [hunk 1]
```diff
@@ -1,5 +1,11 @@
 import { Component, Input } from '@angular/core';
 
+enum DotIconSize {
```
Comment (alfredo-dotcms) [inline src/app/view/components/_common/dot-icon/dot-icon.component.ts]: shouldn't this be on `_varialbles.scss`

File: src/app/view/components/dot-persona-selected-item/dot-persona-selected-item.component.spec.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { DotAvatarModule } from '@components/_common/dot-avatar/dot-avatar.modul
 import { By } from '@angular/platform-browser';
 import { mockDotPersona } from '@tests/dot-persona.mock';
 
-describe('DotPersonaDropdownSelectorComponent', () => {
+fdescribe('DotPersonaDropdownSelectorComponent', () => {
```
Comment (alfredo-dotcms) [inline src/app/view/components/dot-persona-selected-item/dot-persona-selected-item.component.spec.ts]: remove

File: src/app/view/components/_common/dot-icon/dot-icon.component.ts (hunk 1 above)
Comment (fmontes) [reply src/app/view/components/_common/dot-icon/dot-icon.component.ts]: Is a enum, can't be in scss code.

Comment (alfredo-dotcms) [reply src/app/view/components/_common/dot-icon/dot-icon.component.ts]: image

shouldn't it match sizes with scss?

**Pull request merged**